### PR TITLE
thrift 0.13.0

### PR DESCRIPTION
**Pull request by sonerokur**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by EricFromCanada** [comment]
Just needs revision bumps for `apache-arrow` and `osquery`.

**Comment by sonerokur** [comment]
This is my first contribution to homebrew. Can you help me with how to do that ?

File: Formula/apache-arrow.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class ApacheArrow < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.15.0/apache-arrow-0.15.0.tar.gz"
   sha256 "d1072d8c4bf9166949f4b722a89350a88b7c8912f51642a5d52283448acdfd58"
   head "https://github.com/apache/arrow.git"
+  revision 1
```
Comment (EricFromCanada) [inline Formula/apache-arrow.rb]: According to `brew audit apache-arrow`, this line should come before the `head` line.

Comment (sonerokur) [reply Formula/apache-arrow.rb]: Changed the revision line

**Comment by chenrui333** [comment]
is there anyway that you can rebase the of PR to master instead?

**Comment by sonerokur** [comment]
I tried to rebase. Is it ok ?

**Comment by chenrui333** [comment]
Looks good, let's see how it goes!

Thanks!

**Comment by sonerokur** [comment]
Well, this is my first contribution to an open-source project. 

Thank you for your support and understanding.

**Comment by chenrui333** [comment]
Thanks a lot for your interest to contribute on this project!!

**Comment by fxcoudert** [comment]
- `osquery` needs a revision bump
- Please squash into one commit per formula (one for `thrift` and its alias; one for each revision bump)

**Comment by mpercy** [comment]
It looks like arrow has already been bumped. Can someone please try squashing and rebasing this patch?

**Comment by sonerokur** [comment]
All checks have passed. Let's roll it!

**Comment by Bo98** [comment]
Thanks for your first pull request @sonerokur! Merged now.

**Pull request closed without merging**